doc/news/_preparation_next_release.md

```diff
@@ -344,6 +345,8 @@ you up to date with the multi-language support provided by Elektra.
 - Release notes now use git's union merge driver. _(Dominic Jäger)_
 - Please remove me. I'm only here for the build server. _(Dominic Jäger)_
 - I'm only here for the build server. This PR contains only fix ups. _(Dominic Jäger)_
+- Please remove me. I'm only here for the build server. _(Dominic Jäger)_
```

Comment:
Is there any advantage in including this? This definitely increases the chances of merge conflicts.

Reply:
I thought each PR needs a change in this file?

Reply:
Yes, but only one. In your PR there is also another line added. (- Make `kdb editor` use the new merge library. _(Dominic Jäger)_)